src/sections/About.vue

```diff
@@ -20,21 +20,30 @@
           our projects to completion! We are also lucky enough to have sponsors which help provide
           our teams with necessary resources, like servers, to help us achieve our goals.
         </p>
+
+        <ClubSocialsLinks :links="socials" />
       </div>
     </div>
   </div>
 </template>
 
 <script lang="ts">
 import Vue from 'vue';
+import ClubSocialsLinks from '@/components/links/ClubSocialsLinks.vue';
+
```

Comment:
Nit: whitespace or is it your code formatter?

Reply:
kind of intentional - i generally clump my imports into groups when there are > 3 of them. not very well reasoned though, so I suppose theres no reason for the whitespace 🤷‍♂️will remove